model/lib/status-history-log.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var memoize          = require('memoizee/plain')
+  , defineStringLine = require('dbjs-ext/string/string-line');
+
+module.exports = memoize(function (db) {
+	var StringLine = defineStringLine(db);
+
+	return db.Object.extend('StatusHistoryLog', {
```

Comment:
Maybe let's name it `StatusHistoryItem`, it should not be confused with `statusLog` which is similar but different